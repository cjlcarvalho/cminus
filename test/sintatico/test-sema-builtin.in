void main(void)
{
    input();
    println(10);
}
