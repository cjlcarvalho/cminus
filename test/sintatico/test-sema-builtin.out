[program 
  [fun-declaration 
    [void]
    [main]
    [params]
    [compound-stmt 
      [call
        [input]
        [args]
      ]
      [call
        [println]
        [args [10]]
      ]
    ]
  ]
]
